envi/archs/i386/emu.py

```diff
@@ -761,6 +761,19 @@ def i_div(self, op):
             self.setRegister(REG_EAX, quot)
             self.setRegister(REG_EDX, rem)
 
+        elif oper.tsize == 8:
```

Comment:
@invisig0th i'm guessing we should unify the REG_* constants on AMD64.  in Amd64Emultator, the 32-bit DIV instruction is using the 64-bit registers (because it's using i386's REG_EAX, which is 0...  which, in Amd64 is 64-bit RAX.  if we unify, in 32-bit mode, REG_RAX will be 0x200000, which will always be the 32-bit version of the register, no matter what mode we're in.

@williballenthin @mr-tz , the reason we use REG_FOO instead of getRegisterByName() is because it's computational overhead is less.  while a couple function calls are seemingly insignificant for the ease of use, over 10billion instructions it is quite noticeable.